### ISSUE TITLE
Combine Saint Petersburg and Leningrad

### ISSUE THREAD

**Issue by trescube** [whosonfirst-data/whosonfirst-data#948]
My Russian history is a bit rusty but I think [Saint Petersburg](https://whosonfirst.mapzen.com/spelunker/id/102008123/) and [Leningrad](https://whosonfirst.mapzen.com/spelunker/id/102008119/) are the same thing but the polygons between the two are distinct.

**Comment by thisisaaronland** [comment]
Well, one should probably supersede the other but they should _not_ be combined.

**Comment by trescube** [comment]
Whatever works!

**Comment by stepps00** [comment]
St Petersburg is technically a "federal city" and the boundaries we'd think of as "St Petersburg" are actually represented in [this region](https://whosonfirst.mapzen.com/spelunker/id/85688069/) record. 

The region ("federal city") is divided into eighteen separate districts ([source](https://en.wikipedia.org/wiki/Administrative_divisions_of_Saint_Petersburg)), which do not include either the Saint Petersburg or Leningrad localities linked above.

I need to investigate a bit more, but this will probably mean:

- Adding a `wof:placetype_alt` to the `region` record
- Marking some of the region's descendant localities as non-current
- Importing or verifying that we have the eighteen districts of St Petersburg

**Comment by dianashk** [comment]
What's the status on this? It's a pretty glaring mistake and we've had folks write into support about it recently. Would be great to prioritize this. Thanks!

**Comment by stepps00** [comment]
@dianashk - I'll take a look at St. Petersburg this afternoon and update this issue.

**Comment by Chypa74** [comment]
@stepps00 Please don't forget about it. Because this city was renamed from Leningrad to St.Petersburg in 1991)

**Comment by thisisaaronland** [comment]
These two records should _not_ be combined.

One of them should be marked as `mz:is_current=1` and the other `=0`. The latter record should be given an `edtf:cessation` and be superseded by the former.

**Comment by stepps00** [comment]
Hi @Chypa74 - this has been merged and will be available in the next Mapzen Search build, presumably next week.

**Comment by Chypa74** [comment]
Hi @stepps00 this is great news. Thank you

**Comment by Chypa74** [comment]
I will be very happy, if you tell me when the new build will be in production.

**Comment by nvkelso** [comment]
That's a question for @dianashk @orangejulius @trescube

**Comment by stepps00** [comment]
@Chypa74 - there is a Mapzen Search build happening this week, I will update this issue once it is complete.

**Comment by orangejulius** [comment]
We have a new build coming out today in fact, and I just confirmed that Leningrad, Russia is no longer present :) Stay tuned.

**Comment by Chypa74** [comment]
@stepps00 @nvkelso Thank you. You are the best. I really like to use yours geocoder, it is very fast and  have up-to-date information
